maven/maven-quiz.md

```diff
@@ -52,16 +52,10 @@
 #### Q8. Which command is used to run the clean lifecycle followed by verify, install, and package with Maven?
 
 - [ ] `mvn clean install package`
```

Comment:
answer should be "mvn clean verify install package"

Reply:
agree, chatgpt said to me the same 
![image](https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/assets/13170022/6cb24179-5f71-476a-96f1-64b6d0547057)

Reply:
Thanks, @Ebazhanov @haxkd. making necessary changes